test/run

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+#
+# Test the MongoDB image.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+set -exo nounset
+shopt -s nullglob
+
+. test/common-lib
+. test/lib
+
+TEST_LIST="\
+ctest_container_creation
+ctest_configuration
+ctest_general
+ctest_change_password
+ctest_mount_config
+ctest_doc_content
+ctest_local_replication
+ctest_s2i"
+
+VERSION_NUM=$(get_version_number $VERSION)
+if test $VERSION_NUM -ge 30; then
+    TEST_LIST="${TEST_LIST}
+ctest_WT_cache"
+fi
+
+test $# -eq 1 -a "${1-}" == --list && exit 0
+
+CID_FILE_DIR=$(mktemp --suffix=mongodb_test_cidfiles -d)
+
+TEST_DIR="$(readlink -zf $(dirname "${BASH_SOURCE[0]}"))"
+S2I_ARGS="--force-pull=false "
+
+function cleanup() {
+  local network_name="mongodb-replset-$$"
+  ct_cleanup
+  ! docker network ls | grep -q ${network_name} || docker network rm ${network_name}
+}
+trap cleanup EXIT SIGINT
+
+function test_mongo() {
+    echo "  Testing MongoDB"
+    if [ -v ADMIN_PASS ]; then
+        echo "  Testing Admin user privileges"
+        mongo_admin_cmd "db=db.getSiblingDB('${DB}');db.dropUser('${USER}');"
+        mongo_admin_cmd "db=db.getSiblingDB('${DB}');db.createUser({user:'${USER}',pwd:'${PASS}',roles:['readWrite','userAdmin','dbAdmin']});"
+        mongo_admin_cmd "db=db.getSiblingDB('${DB}');db.testData.insert({x:0});"
+        mongo_cmd "db.createUser({user:'test_user2',pwd:'test_password2',roles:['readWrite']});"
+    fi
+    echo "  Testing user privileges"
+    mongo_cmd "db.testData.insert({ y : 1 });"
+    mongo_cmd "db.testData.insert({ z : 2 });"
+    mongo_cmd "db.testData.find().forEach(printjson);"
+    mongo_cmd "db.testData.count();"
+    mongo_cmd "db.testData.drop();"
+    mongo_cmd "db.dropDatabase();"
+    echo "  Success!"
+}
+
+function ctest_configuration() {
+    echo "  Testing image configuration settings"
+    test_config_option MONGODB_QUIET true "quiet: true"
+    if test $VERSION_NUM -le 30; then
+        test_config_option MONGODB_PREALLOC false "preallocDataFiles: false"
+        test_config_option MONGODB_NOPREALLOC true "preallocDataFiles: false"
+        test_config_option MONGODB_SMALLFILES true "smallFiles: true"
+    fi
+    echo "  Success!"
+}
+
+function ctest_mount_config() {
+    local name="mount_config"
+    echo "  Testing config file mount"
+    local database='db'
+    local user='user'
+    local password='password'
+    local admin_password='adminPassword'
+
+    local volume_dir
+    local config_content
+    volume_dir=`mktemp -d --tmpdir mongodb-testdata.XXXXX`
+    chmod a+rwx ${volume_dir}
+    config_file=$volume_dir/mongod.conf
+    config_content="dbpath=/var/lib/mongodb/dbpath
+unixSocketPrefix = /var/lib/mongodb"
+    if test $VERSION_NUM -le 30; then
+    config_content="$config_content
+smallfiles = true
+noprealloc = true"
+    fi
+    echo $config_content > $config_file
```

Comment:
`"` is missing (`echo "$config_content"`). Without it all content is in one line.

Reply:
Fixed, thanks!